ironic_bmo_configmap.yaml

```diff
@@ -0,0 +1,15 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ironic-bmo-configmap
+data:
+  http_port: "6180"
+  provisioning_interface: "ens3"
+  provisioning_ip: "172.22.0.3/24"
+  dhcp_range: "172.22.0.10,172.22.0.100"
+  deploy_kernel_url: "http://172.22.0.3:6180/images/ironic-python-agent.kernel"
+  deploy_ramdisk_url: "http://172.22.0.3:6180/images/ironic-python-agent.initramfs"
+  ironic_endpoint: "http://172.22.0.3:6385/v1/"
+  ironic_inspector_endpoint: "http://172.22.0.3:5050/v1/"
+  cache_url: "http://192.168.111.1/images"
+  rhcos_image_url: "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190520.0/"
```

Comment:
This needs to be the placeholder for sed e.g `__RHCOS_IMAGE_URL__` not this old image URL

Reply:
OK. fixed.